### PR TITLE
Add German translation

### PR DESCRIPTION
**Pull request by fhemberger**

### CODE REVIEW

**Comment by fhemberger** [comment]
Already reviewed by @iojs/iojs-de.

File: de/index.html [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+			<li>Wird immer abwärtskompatibel bleiben.</li>
+			<li>Neue APIs können auf ES6/7 fokussiert sein,<br>ohne alte Module und APIs zu beeinflussen.</li>
+			<li>Trennt die Arbeit an neuen APIs von der Weiterentwicklung der aktuellen Plattform.</li>
+		</ul>
+		<br>
+		<p class="note">NG-bezogene Diskussionen finden im <a href="https://github.com/iojs/ng">NG-Repository</a> statt.</p>
+	</div></section>
+
+	<section class="slide"><div>
+		<h2>Debugging & Tracing</h2>
```
Comment (Andersos) [inline de/index.html]: Shouldn't & be replaced by &amp; ?

Comment (fhemberger) [inline de/index.html]: It wasn't in all the other translations, so I kept it that way. But yes, I'll add it.

**Comment by fibric** [comment]
Can we merge this PR now?

Because on roadmap.iojs.org/ the link to 'Deutsch' version points http://roadmap.iojs.org/de/ which currently does not exists in gh-pages.

**Comment by fhemberger** [comment]
@mikeal *bump* As @fibric already said, the German version links are currently broken in production because this PR missing.

**Comment by fhemberger** [comment]
Great, thanks!

**Pull request merged**